db/migrate/20160302203922_remove_is_datacenter_from_ems_folder.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveIsDatacenterFromEmsFolder < ActiveRecord::Migration[5.0]
+  def up
+    # Go through all existing EmsFolder records and set :type => Datacenter
+    # if the is_datacenter column is true before deleting it
+    EmsFolder.all.each do |folder|
```

Comment:
https://github.com/ManageIQ/manageiq/blob/ee3764127370397f1a6027a9c2fe7a0a539d93f6/db/migrate/20140121213913_split_widget_set_name_to_three_columns.rb#L2

Ah, you put it in the spec/migration... put that here instead and migration_stub will find it.